spec/imsc-hrm.html

```diff
@@ -189,7 +186,7 @@
 <body>
   <section id='abstract'>
     <p>This specification specifies an Hypothetical Render Model (HRM) that constrains the presentation complexity of
-    documents that conform to any of the TTML Profiles for Internet Media Subtitles and Captions ([[IMSC]]).</p>
+    documents that conform to the Text Profile specified in Internet Media Subtitles and Captions ([[IMSC]]).</p>
```

Comment:
We lost the sense of support across versions of IMSC. This essentially duplicates the term definition of "IMSC Document Instance".

```suggestion
    documents that conform to the Text Profiles specified in any edition of Internet Media Subtitles and Captions ([[IMSC]]).</p>
```